[DO NOT MERGE] DOCS-12770 lifecycle states (#166)
mulesoft/docs-exchange: modules/ROOT/pages/publish-an-http-api.adoc

:imagesdir: ../assets/images/
:page-aliases: to-share-an-http-api.adoc

The HTTP API lets Anypoint Exchange users specify a placeholder for an endpoint. After you create the HTTP API endpoint, you can optionally manage the endpoint with API Manager.

The HTTP API asset describes what the API endpoint does and how users can connect the API to their programs.

In API Manager, users import the API from Exchange and can then associate proxies and manage policies and SLAs for the API.

. In a Exchange, click *Publish new asset*.
. Enter a name for the API
. Choose an *HTTP API* from the *Asset types* drop-down menu.
. Click *Publish*.

> . In the asset view, click *Edit documentation* to provide a description.

Reviewer: To edit the description of an asset what they should click is the pencil icon

> . Click *Save* and *Publish*.

Reviewer: Clicking Save is not needed if the final goal is to publish

:imagesdir: ../assets/images/
:page-aliases: to-share-an-http-api.adoc

> The HTTP API lets Anypoint Exchange users specify a placeholder for an endpoint. After you create the HTTP API endpoint, you can optionally manage the endpoint with API Manager.

Reviewer 2: ```suggestion
The HTTP API enables Anypoint Exchange users to specify a placeholder for an endpoint. After you create the HTTP API endpoint, you can optionally manage the endpoint with API Manager.
```

The HTTP API asset describes what the API endpoint does and how users can connect the API to their programs.

In API Manager, users import the API from Exchange and can then associate proxies and manage policies and SLAs for the API.

> . In a Exchange, click *Publish new asset*.

Reviewer 2: ```suggestion
. In Exchange, click *Publish new asset*.
```

> . Enter a name for the API

Reviewer 2: ```suggestion
. Enter a name for the API.
```

> . Choose an *HTTP API* from the *Asset types* drop-down menu.

Reviewer 2: ```suggestion
. Choose an *HTTP API* value from the *Asset types* drop-down menu.
```

. Click *Publish*.

> . In the asset view, click the pencil icon to provide a description and click the check icon.

Reviewer 2: ```suggestion
. In the asset view, click the pencil icon to provide a description, and then click the checkmark icon.
```
We should show rather than describe the "check" icon, or rename it "checkmark" to be just a little more clear.
